ProcessLib/ProcessVariable.cpp

```diff
@@ -34,6 +34,9 @@ ProcessVariable::ProcessVariable(
       _bc_builder(new BoundaryConditionBuilder())
 {
     DBUG("Constructing process variable %s", _name.c_str());
+    //! \ogs_file_param{prj__process_variables__process_variable__order}
+    auto opt_shapefunction_order = config.getConfigParameterOptional<int>("order");
+    _shapefunction_order = (opt_shapefunction_order ? opt_shapefunction_order.get() : 1);
```

Comment:
Need an assert somewhere such that this input order is not larger than the order of element of the mesh.